javascript-carousel/main.js

```diff
@@ -0,0 +1,79 @@
+var intervalID = setInterval(cycleImg, 3000);
+
+var imgGallery = ['001', '004', '007', '025', '039'];
+var $img = document.querySelector('img');
+var counter = 0;
+var imgCount = imgGallery.length - 1;
+var $progressDot = document.querySelectorAll('.progress-dot');
+var $imgCarousel = document.querySelector('.image-carousel');
+var $progressBar = document.querySelector('.progress-bar');
+
+function cycleImg() {
+  counter++;
+  $img.src = 'http://127.0.0.1:5500/javascript-carousel/images/' + imgGallery[counter] + '.png';
```

Comment:
I wouldn't use the full absolute URL for image `src`s, as it'll break apps that need to work in more than one environment.